demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -165,6 +166,19 @@ def ignored_errors(self) -> list:
         except:  # noqa: E722
             return []
 
+    def ignored_errors_related_files(self, file_path: str) -> list:
+        try:
+            return (
+                list(
+                    self.ignored_errors_dict.get(  # type: ignore
+                        f"file:{(Path(file_path)).name}", []
+                    ).items()
+                )[0][1].split(",")
+                or []
+            )
+        except:  # noqa: E722
```

Comment:
same comment as other func, why do we need the same func?

Reply:
It's not exactly the same func.
For pack we try to get the ignored_errors_dict from a different location.
Also, when we try to get the pack ignored errors we can use self.path like in the rest of the content items.